recipes/libaom-av1/all/conandata.yml

```diff
@@ -15,6 +18,8 @@ sources:
     url: "https://storage.googleapis.com/aom-releases/libaom-2.0.1.tar.gz"
     sha256: "a0cff299621e2ef885aba219c498fa39a7d9a7ddf47585a118fd66c64ad1b312"
 patches:
+  "3.5.0":
```

Comment:
Can you add the patch metadata fields for this patch? The fields are documented [here](https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields).